src/components/user/EditUser.jsx

```diff
@@ -1,24 +1,20 @@
 import React from "react";
-import { useState, useEffect } from "react";
-import { Link, useNavigate } from "react-router-dom";
+import useState from "react";
 import axios from "axios";
-
 import styled from "styled-components";
-import { Checkbox } from "@mantine/core";
-import { CloseButton } from "@mantine/core";
-import { List } from "@mantine/core";
-import { Popover, Button, TextInput } from "@mantine/core";
-import { Modal, Group, Text } from "@mantine/core";
+import HelloUser from "./HelloUser";
 import { useDisclosure } from "@mantine/hooks";
 import { PasswordInput } from "@mantine/core";
+import { Button, TextInput } from "@mantine/core";
+import { Link, useNavigate } from "react-router-dom";
 
 const Wrapper = styled.form`
```

Comment:
Wrapperの意味を考えて見てほしいです！！
（調べると色々出てきます🖐）

ここはformのhtmlタグに対してスタイルを再定義しているので、Formであることを明確にしたほうが良いです！
[ここの箇所](https://github.com/daiki-shiroma/auth-app-client/pull/2/files#diff-a9e73f1df9bc2a6de37c91007fe094204ea141c545cd52deb4727b1832fa65a7R10)も同様


```suggestion
const UserEditForm = styled.form`
```

下記の箇所の使い方はOKだと思います！
https://github.com/daiki-shiroma/auth-app-client/pull/2/files#diff-8bc8933bf9f0a1dece53f0af6a1cf667e51f3286cc1d8aacebe87ff459e2960bR9